integrations/woopra/lib/index.js

```diff
@@ -139,35 +147,9 @@ Woopra.prototype.identify = function(identify) {
 
 Woopra.prototype.track = function(track) {
   setContext(track);
-  window.woopra.track(track.event(), stringifyNested(track.properties()));
+  window.woopra.track(track.event(), track.properties());
 };
 
-/**
- * Stringify nested objects.
- *
- * Undocumented aspect of Woopra's API, but apparently required. Breaks
- * on `Completed Order` `properties.products`.
- *
- * @param {Object} obj
- * @return {Object}
- * @api private
- */
-
-function stringifyNested(obj) {
-  return foldl(
```

Comment:
Can you also remove the `@ndhoule/foldl` import now that we don't need to use it?

Reply:
Sure